File: FinalProject.WebForm/Models/Guest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FinalProject.WebForm.Models
{
    public class Guest
    {
        public int GuestId { get; set; }

        public string GuestName { get; set; }

        public string Email { get; set; }

        public string PhoneNumber { get; set; }
    }
}

File: automobil/lib/models/guest.dart
class Guest {
  final int? guestId;
  final String guestName;
  final String email;
  final String phoneNumber;

  Guest({
    this.guestId,
    required this.guestName,
    required this.email,
    required this.phoneNumber,
  });

  factory Guest.fromJson(Map<String, dynamic> json) {
    return Guest(
      guestId: json['GuestId'],
      guestName: json['GuestName'],
      email: json['Email'],
      phoneNumber: json['PhoneNumber'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'GuestId': guestId,
      'GuestName': guestName,
      'Email': email,
      'PhoneNumber': phoneNumber,
    };
  }
}
